### DISCUSSION TITLE
How to put condition expression while doing registry search.

### DISCUSSION THREAD

**Discussion by radz1** [wixtoolset/issues#6356] [Questions]
**How to put condition expression while doing registry search.** 

I want to search registry only when **IsPreviousVersionFootprintExist** exist and LOGFOLDER variable is empty.
If LOGFOLDER variable is passed from the command line, we do not want to search the registry.

i tried condition as below but not working - can not parse.
 **Condition="IsPreviousVersionFootprintExist AND LOGFOLDER=''"**

` <util:RegistrySearch Id="IsPreVerFootprintExist" Variable="IsPreviousVersionFootprintExist" Root="HKCU" Key="Settings" Result="exists" Win64="no" />`

`<util:RegistrySearch Root="HKCU" Key="Settings" Result="value" Variable="LOGFOLDER" Win64="no" Format="raw" Value="KeylogFolder" Condition="IsPreviousVersionFootprintExist AND LOGFOLDER=''"/>`

**Comment by robmen** [comment]
Burn does not automatically convert command-line arguments into `Variables`. Your BA needs to do that. If you use wixstdba then the [bal:Overridable](https://wixtoolset.org/documentation/manual/v3/xsd/wix/variable.html) extension attribute can be used to convert command-line args to `Variables`
